src/XDGData/org.freecadweb.FreeCAD.xml

```diff
@@ -4,6 +4,6 @@
     <!-- <sub-class-of type="application/zip"/> -->
     <comment>FreeCAD document files</comment>
     <glob pattern="*.fcstd"/>
-    <icon name="org.freecadweb.FreeCAD"/>
+    <generic-icon name="org.freecadweb.FreeCAD"/>
```

Comment:
shouldn't we be using application-x-extension-fcstd as an icon?
https://github.com/FreeCAD/FreeCAD/blob/10a78d91af1e97b8c4ae3b3ec14dc7da301df854/src/Gui/CMakeLists.txt#L1290
also I don't see why generic-icon would be more appropriate?